bundles/org.openhab.binding.openwebnet/README.md

```diff
@@ -314,6 +314,11 @@ In order to activate one of them you have to use two different channels:
 
 Example: if you want to activate SCENARIO number 9 on the thermo Central Unit you have to set `mode` = `SCENARIO` and `scenarioProgram` = `9`.
 
+### Energy Channels
+
+`energyToday`  and `energyThisMonth` channel updates depend on F520/F521  measurements. As a consequence, 
+if the meter doesn't register any  variation since the last reading the value is not updated.
+
```

Comment:
are we sure this is useful information? In fact it's always the case in OH (or any home automation) that if a device/sensor does not send updates, then the thing/items will not be updated...

Reply:
The bulk is that the meter has it's own set updating frequency which diverge from energyRefreshPeriod. I read the OWN documentation, it seems to be the possibility to set  the meter's  updating frequency  but the doc is  very grim. In my case the upting period is one hour. So  you are right it is no use!
I will remove it